upstream-community-operators/federatorai/federatorai.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,484 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: federatorai.v0.0.1
+  namespace: placeholder
+  annotations:
+    capabilities: Basic Install
+    categories: OpenShift Optional
```

Comment:
I'm not sure if this operator fits under the OpenShift Optional component category. Additional categories can be found here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories
If you believe this category best fits your operator, you are welcome to leave it as is :)

Reply:
Haha we had added "AI/Machine Learning" category to it.